towncrier/newsfragments/1221.enhancement

```diff
@@ -0,0 +1 @@
+Dovecot: Delete obsolete data volume
```

Comment:
We usually don't use the `.enhancement` extension for [towncrier](https://mailu.io/1.7/contributors/guide.html#changelog). I'm not sure if towncrier supports custom extensions? Maybe `.removal` or `.misc` fits better.

Reply:
Changed to `.removal`